styleguide/source/_patterns/03-organisms/tools-block.twig

```diff
@@ -4,7 +4,7 @@
     {% include "@atoms/heading.twig" %}
   {% endblock %}
 
-  {% block topicTools %}
+  {% block tools %}
```

Comment:
This organism should also include the learn more organism

Reply:
done